articles/cosmos-db/mongodb-feature-support.md

```diff
@@ -289,7 +294,11 @@ $all | ```{ "Location.coordinates": { $all: [-121.758, 46.87] } }``` |
 $elemMatch | ```{ "Location.coordinates": { $elemMatch: {  $lt: 0 } } }``` |  
 $size | ```{ "Location.coordinates": { $size: 2 } }``` | 
 $comment |  ```{ "Location.coordinates": { $elemMatch: {  $lt: 0 } }, $comment: "Negative values"}``` | 
-$text |  | Not supported. Use $regex instead 
+$text |  | Not supported. Use $regex instead.
+
+## Unsupported operators
+
+The ```$where``` and the ```$eval``` operators are not supported by Cosmos DB and there are no plans for implementation.
```

Comment:
I would remove "there are no plans for implementation" to be more user-friendly. you can just say they are currently not supported.

Reply:
Would clarifying that they've been deprecated by MongoDB be clearer?